assets/performanceprofile/scripts/low-latency-hooks.sh

```diff
@@ -1,35 +1,38 @@
 #!/usr/bin/env bash
 
+BIN_PATH="/usr/bin"
+SBIN_PATH="/usr/sbin"
```

Comment:
I'd rather use variables to encode the paths of all the binaries we depend on:
```
JQ="/usr/bin/jq"
IP="/usr/sbin/ip"
...
```
The extra benefit is we have all the deps explicit.
We should do this for all the commands we use which are not shell builtins
Even better, we should replace these scripts with real binaries in the future.